test/test_logistic_regression.py

```diff
@@ -11,7 +11,7 @@ class TestLogisticRegression(unittest.TestCase):
     """
     n = 16
     k = 2
-    N = 12000
+    N = 500
```

Comment:
Could you decrease n to 8 and set N to 2**8? This would be more consistent to the other tests.